FAQ.md

```diff
@@ -0,0 +1,151 @@
+
+### Do I have to pay to use this list?
+No. Also, the maintenance of this service is self-financed.
+
+### Are you planning to include a VOD of TV shows broadcasted on this playlist?
+No. This is not our purpose.
+
+Based on [#175](https://github.com/iptv-org/iptv/issues/175), [#3290](https://github.com/iptv-org/iptv/discussions/3290), [#7852](https://github.com/iptv-org/iptv/discussions/7852).
+
+## Channels
+### Does the channel need to be publicly accessible to be included in the playlists?
+Yes, we need the links to be publicly accessible like FTA packages (free-to-air), online services or official websites. We recommend tagging [Geo-blocked] if it is only accessible in one or few countries.
+
+Based on [#480](https://github.com/iptv-org/iptv/issues/480).
+
+### Can I add a FTA channel that broadcasts PPV events or pay programming?
+Maybe. Consider if the country where it is broadcasted allows fair use. Owners can ban the broadcast of PPV events, and even issue a DMCA takedown. Check the [blocklist](https://github.com/iptv-org/database/blob/master/data/blocklist.csv) for more info about specific events, and this [counternotice](https://github.com/github/dmca/blob/3ce1c9402c6b7e46dcbbea6ff866d6a44e75c72a/2020/10/2020-10-08-dfl-counternotice.md) indicating that a DMCA takedown is invalid if only one FTA channel infringes copyright.
+
+Please remember that by sharing a link to a FTA channel (or more), you are **indexing** the link, not **restreaming** its content. This subtility makes most of the DMCA takedowns false and invalid under law.
+A good example of this would be [Perfect 10 v. Google](https://www.eff.org/fr/cases/perfect-10-v-google). 
+Some [valid takedowns](https://github.com/iptv-org/iptv/issues/6486) do occur when a Pull Request is made to notify us that we should not index their channels.
+
+Based on [#6350](https://github.com/iptv-org/iptv/issues/6350).
+
+### I've created my channel and I want to add it to this playlist. What should I do?
+We are a community initiative and you don't need to pay anything to stay on this playlist. Feel free to create a channel with few resources:
+* Camera with a tripod (recommended for your own studio).
+* A PC with Internet, video editor and recording/broadcasting app under RTMP protocol (e.g. OBS).
+* And a live streaming service with multibitrate (if possible).
+
+After you have your m3u8 link, add your channel information onto the [Database](https://github.com/iptv-org/database) repository, and submit an [issue](https://github.com/iptv-org/iptv/issues/new) to propose it to us.
+
+Note: There are cases of broadcasting a copy of third-party content (released previously in public) with the purpose of promoting, educating or commenting. This is known as fair use (also called "fair dealing" or "copyright exception") and should be important when broadcasting without problems on your TV channel.
+
+### Is it possible to limit the broadcasting of my channel to my country only?
```

Comment:
I can't find the origin of the question. At the link, a man asks to remove from playlists channels blocked in his country.

Reply:
One example is about a TV news channel that use to third party material for commentary. Do you understand what I'm saying?

Reply:
I'm sorry, I thought you wanted to collect the actual frequently asked questions from the whole repository together with the answers and just put them in one file, that's the only reason I asked for the source.

Since it's just your own q&a you don't need to add any links of course.

Reply:
This question is more for those who wish to create their own television station. As the creators tend to be more from Latin America (I guess, for online streaming), I doubt that they will ask questions in English for Github (just a channel from Argentina as evidence). So I found it useful.

Reply:
> This question is more for those who wish to create their own television station. As the creators tend to be more from Latin America (I guess, for online streaming), I doubt that they will ask questions in English for Github (just a channel from Argentina as evidence). So I found it useful.

Don't hesitate, if you're onto translating, to make one Spanish-translated FAQ. I could make a French one

Reply:
This PR must be finished first, and then the translation must be done. We need to complete the community consensus to continue.